lib/msal-browser/src/cache/BrowserCacheManager.ts

```diff
@@ -400,26 +400,48 @@ export class BrowserCacheManager extends CacheManager {
      * Gets the active account
      */
     getActiveAccount(): AccountInfo | null {
-        const activeAccountIdKey = this.generateCacheKey(PersistentCacheKeys.ACTIVE_ACCOUNT);
-        const activeAccountId = this.browserStorage.getItem(activeAccountIdKey);
-        if (!activeAccountId) {
+        const activeAccountKeyHomeLocalIds = this.generateCacheKey(PersistentCacheKeys.ACTIVE_ACCOUNT_HOME_LOCAL_IDS);
+        const activeAccountValueHomeLocalIds = this.getItem(activeAccountKeyHomeLocalIds);
+        if (!activeAccountValueHomeLocalIds) { 
```

Comment:
Can we add some `Trace` level logs in this function to help us debug what's going on?

Reply:
Fixed